app/run.py

```diff
@@ -26,10 +26,23 @@ def dimensions():
 @app.route('/results', methods=['POST'])
 def results():
     """
-    Performs most of the logic for paint calculations
+    Calculates the result and renders the page
```

Comment:
Separating this allows it to test the logic to calculate the needed gallons from the rendering part